diff --git a/Source/Core/FontEngineDefault/FreeTypeInterface.cpp b/Source/Core/FontEngineDefault/FreeTypeInterface.cpp
index f162dd2..6d49751 100644
--- a/Source/Core/FontEngineDefault/FreeTypeInterface.cpp
+++ b/Source/Core/FontEngineDefault/FreeTypeInterface.cpp
@@ -280,7 +280,7 @@ static bool BuildGlyph(FT_Face ft_face, const Character character, FontGlyphMap&
 	if (index == 0)
 		return false;
 
-	FT_Error error = FT_Load_Glyph(ft_face, index, FT_LOAD_COLOR);
+	FT_Error error = FT_Load_Glyph(ft_face, index, FT_LOAD_COLOR | FT_LOAD_NO_HINTING);
 	if (error != 0)
 	{
 		Log::Message(Log::LT_WARNING, "Unable to load glyph for character '%u' on the font face '%s %s'; error code: %d.", (unsigned int)character,
